exporters/prometheus/exporter.go

```diff
@@ -660,7 +671,11 @@ func addExemplars[N int64 | float64](
 func attributesToLabels(attrs []attribute.KeyValue, labelNamer otlptranslator.LabelNamer) prometheus.Labels {
 	labels := make(map[string]string)
 	for _, attr := range attrs {
-		labels[labelNamer.Build(string(attr.Key))] = attr.Value.Emit()
+		name, err := labelNamer.Build(string(attr.Key))
+		if err != nil {
+			continue
```

Comment:
I think we need to propagate the label back here and skip the series.  Just skipping a label will likely cause duplicate timeseries.

Reply:
ah, yes.  Didn't realize this was for exemplar labels.  In that case, it is probably OK to just skip the exemplar label.